### PR TITLE
Started work on items

### PR DESCRIPTION
**Pull request by LagoLunatic**
Got a few functions to 100% and started defining structs.

### CODE REVIEW

File: include/d/actor/d_a_itembase.h [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+public:
+    /* 0x294 */ struct request_of_phase_process_class mPhs;
+    /* 0x29C */ struct J3DModel* mModel;
+    /* 0x2A0 */ struct J3DModel* mModelArrow[2]; // Extra models for arrow bundles
+    /* 0x2A8 */ struct mDoExt_btkAnm* mBtkAnm1;
+    /* 0x2AC */ struct mDoExt_btkAnm* mBtkAnm2;
+    /* 0x2B0 */ struct mDoExt_brkAnm* mBrkAnm1;
+    /* 0x2B4 */ struct mDoExt_brkAnm* mBrkAnm2;
+    /* 0x2B8 */ struct mDoExt_bckAnm* mBckAnm;
```
Comment (TakaRikka) [inline include/d/actor/d_a_itembase.h:66]: it's probably better to use C++ style forward declares here. and where not possible, just include the header (phase req is c_phase.h iirc)

**Comment by magcius** [comment]
Thanks! I might move some of the item_static / itembase files around, I meant for the actor/ folder to only contain the processes, though d_a_* picked up a bit too much.

File: include/d/actor/d_a_itembase.h (hunk 1 above)
Comment (LagoLunatic) [reply include/d/actor/d_a_itembase.h:66]: Oh whoops, I forgot to remove the struct keywords when copying from ghidra.

**Comment by LagoLunatic** [comment]
> Thanks! I might move some of the item_static / itembase files around, I meant for the actor/ folder to only contain the processes, though d_a_* picked up a bit too much.

Yeah that's fine by me, I wasn't sure what the exact directory structure should be.

**Pull request merged**